src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,5 +1,7 @@
 package mate.academy.service;
 
+import mate.academy.model.User;
+
 public class AuthenticationService {
```

Comment:
Create variable for service you depend on to use required methods
```suggestion
public class AuthenticationService {
    private final UserService userService = new UserService();
```